src/storage/metrics.rs

```diff
@@ -617,4 +624,18 @@ lazy_static! {
         &["type"]
     )
     .unwrap();
+
+    pub static ref SCHED_TXN_MEMORY_QUOTA: MemoryQuotaGauge = register_static_int_gauge_vec!(
+        MemoryQuotaGauge,
+        "tikv_scheduler_memory_quota_size",
+        "Statistics of in_use and capacity of scheduler memory quota",
+        &["type"]
+    )
+    .unwrap();
+
+    pub static ref SCHED_TXN_RUNNING_COMMANDS: IntGauge = register_int_gauge!(
```

Comment:
Currently, there is a pending command statistic that is more closely related to memory usage at https://github.com/tikv/tikv/pull/16482/files#diff-fd86c97f137c667fa1222b07fd71d8e369b7ada70a34afc8415d1665ef9ea9e8L407. Is it still necessary to have `running`?

Reply:
The future spawns by a `running` txn command takes about 6184 bytes, I think it's worth to record, also it reflects how busy the scheduler pool is.